subsys/caf/modules/ble_state.c

```diff
@@ -16,7 +16,7 @@
 #include <caf/events/ble_common_event.h>
 
 #ifdef CONFIG_CAF_BLE_USE_LLPM
-#include "sdc_hci_vs.h"
+#include <bluetooth/hci_vs_sdc.h>
 #endif /* CONFIG_CAF_BLE_USE_LLPM */
```

Comment:
Please remove `#ifdef` if no longer needed

Reply:
It is still needed. This header includes sdc_hci_vs.h